--- qoverview.py	2017-07-07 22:19:16.170044502 +0530
+++ qoverview.py	2017-07-07 22:33:12.418643716 +0530
@@ -15,7 +15,7 @@

 tmp_dir = os.path.join(os.environ.get('XDG_RUNTIME_DIR', '/tmp'), 'qoverview')

-KDE_FRAMEWORKS = True
+KDE_FRAMEWORKS = False

 class PythonQMLInterface(QObject):

@@ -68,7 +68,7 @@
 		for entry in self.apps_list:
 			try:
 				if entry['Name'].lower().startswith(search_terms.lower()) and entry['Name'] not in done:
-					results.append([entry['Name'], entry['EntryName'], entry['Icon']])
+					results.append([entry['Name'], entry['EntryName'], entry['IconPath']])
 					done.append(entry['Name'])

 			except KeyError:
@@ -91,7 +91,7 @@

 		for entry in self.apps_list:
 			if entry['EntryName'] in dock_items_list:
-				results.append([entry['Name'], entry['EntryName'], entry['Icon']])
+				results.append([entry['Name'], entry['EntryName'], entry['IconPath']])

 		return results

@@ -102,7 +102,7 @@

 		for index, w_id in enumerate(ids):
 			if wm.get_window_name(w_id) not in [self.uid, 'Desktop — Plasma']:
-				results.append([wm.get_window_name(w_id), w_id, int(w_id, 16)])
+				results.append([wm.get_window_name(w_id), w_id, wm.get_window_screenshot(w_id, str(int(w_id, 16)))])

 		return results

