--- qoverview.py	2017-07-06 12:42:12.331213438 +0530
+++ /tmp/qoverview.py	2017-07-06 12:59:09.084035189 +0530
@@ -103,7 +103,7 @@
 
 		for index, w_id in enumerate(ids):
 			if wm.get_window_name(w_id) not in [self.uid, 'Desktop — Plasma']:
-				results.append([wm.get_window_name(w_id), w_id, int(w_id, 16)])
+				results.append([wm.get_window_name(w_id), w_id, wm.get_window_screenshot(w_id)])
 
 		return results
 
